Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -189,8 +189,7 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
     
     override open func updateContent() {
         if let channel = content.channel {
-            let namer = uiConfig.channelList.channelNamer.init()
-            titleLabel.text = namer.name(for: channel, as: content.currentUserId)
+            titleLabel.text = DefaultChatChannelNamer()(channel, content.currentUserId)
```

Comment:
Why did you make this change? Previously the namer was configurable via `UIConfig` and now it's hardcoded.

Reply:
This is done.